scripts/monthly-server-report.py

```
@@ -0,0 +1,192 @@
+**2. AWS WAF (보안)**
+- 🛡️ **총 요청:** `{total_req:,}건`
+- ✅ **허용됨:** `{waf_data['allowed']:,}건` ({waf_data['allowed'] / total_req * 100:.1f}%)
+- 🚫 **차단됨:** `{waf_data['blocked']:,}건`
+
+**3. AWS Cost (비용)**
+- 💰 **이번 달:** `${curr_cost:.2f}`
+- {cost_emoji} **전월 대비:** `{cost_diff_str}` ({'증가' if cost_diff > 0 else '감소' if cost_diff < 0 else '변동없음'})
+""".strip()
+
+    try:
+        requests.post(DISCORD_WEBHOOK_URL, json={
```

Ruff (0.14.6)

Probable use of `requests` call without timeout

(S113)